ipmininet/iptopo.py

```diff
@@ -116,7 +118,7 @@ def __new__(cls, value, *args, **kwargs):
 
     def __init__(self, o, topo):
         self.topo = topo
-        super(RouterDescription, self).__init__(o)
+        super(RouterDescription, self).__init__()
```

Comment:
Why did the init() parameter 'o' disappear here?

Reply:
Because Python3 raises the following exception if the argument is present:
```
TypeError: object.__init__() takes no parameters
```

Reply:
What are we missing as a functionnality due to that then?

Reply:
None, I think. Python2.7 was already complaining about that in a warning. Python3 enforces it.
This creates a perfectly fine str object.